appveyor.yml

```diff
@@ -68,7 +68,9 @@ install:
   # for obvci_appveyor_python_build_env.cmd
   - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
   # for msinttypes and newer stuff
-  - cmd: conda config --add channels conda-forge
+  # conda-forge may serve outdated versions of certain packages (e.g. conda
+  # itself), so append it to the end of the list.
+  - cmd: conda config --append channels conda-forge
```

Comment:
1. Right now conda-forge is serving an outdated version of conda which is the reason the Appveyor build broke.  I would hope that the official conda channel never serves an outdated version of conda...  I don't think serving outdated versions of other build dependencies is as much of an issue (if any).
2. I could *instead* set the `channel_priority` option documented at https://conda.io/docs/channels.html#managing-conda-channels, but there are other issues with that (see the link).  I don't have a strong opinion though.